rdt-open-reader/src/test/java/io/ona/rdt/robolectric/widget/WidgetFactoryRobolectricTest.java

```diff
@@ -48,4 +48,25 @@ private static Intent getJsonFormActivityIntent() throws JSONException {
         intent.putExtra(JsonFormConstants.JSON_FORM_KEY.JSON, mJSONObject.toString());
         return intent;
     }
+
+    public RDTJsonFormActivity getRDTJsonFormActivityNonStatic() throws JSONException {
```

Comment:
Any reason we need a non-static version?

Reply:
Actually, I need to modify the `step 1` JSONObject which was directly creating under the method  of `getJsonFormActivityIntent` that's why I have created a non-static version of both methods and in addition, added another method called `getJsonObject()` which will return a JSONObject for step 1 and I have override this method in https://github.com/onaio/rdt-standard/blob/475d773834a237dcbcc2e993a0fdb4da4ad8f86c/rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/CovidRepeatingGroupFactoryTest.java

Reply:
Why not keep the same method and call the `getJsonObject()` instead which returns `new JSONObject()` by default and can still be overridden in the new test?

Reply:
I have removed the non-static version and use a different approach.

Reply:
the official doc says `Your fragment shouldn't be dependent on a specific parent activity or fragment.` https://developer.android.com/guide/fragments/test